Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/TrendArrowView.kt

```diff
@@ -2,20 +2,47 @@ package de.rki.coronawarnapp.statistics.ui
 
 import android.content.Context
 import android.util.AttributeSet
-import android.view.View
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import de.rki.coronawarnapp.R
 import de.rki.coronawarnapp.server.protocols.internal.stats.KeyFigureCardOuterClass
-import timber.log.Timber
+import de.rki.coronawarnapp.server.protocols.internal.stats.KeyFigureCardOuterClass.KeyFigure.Trend.DECREASING
+import de.rki.coronawarnapp.server.protocols.internal.stats.KeyFigureCardOuterClass.KeyFigure.Trend.INCREASING
+import de.rki.coronawarnapp.server.protocols.internal.stats.KeyFigureCardOuterClass.KeyFigure.TrendSemantic.NEGATIVE
+import de.rki.coronawarnapp.server.protocols.internal.stats.KeyFigureCardOuterClass.KeyFigure.TrendSemantic.POSITIVE
 
 class TrendArrowView @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
-) : View(context, attrs, defStyleAttr) {
+) : FrameLayout(context, attrs, defStyleAttr) {
+
+    private val imageView: ImageView
+
+    init {
+        LayoutInflater.from(context)
+            .inflate(R.layout.statistics_trend_view, this, true)
+        imageView = findViewById(R.id.trend)
+    }
 
     fun setTrend(
         trend: KeyFigureCardOuterClass.KeyFigure.Trend,
         trendSemantic: KeyFigureCardOuterClass.KeyFigure.TrendSemantic
     ) {
-        Timber.d("setTrend(trend=$trend, trendSemantic=$trendSemantic)")
+        imageView.rotation = when (trend) {
+            INCREASING -> -45F
+            DECREASING -> 45F
+            else -> 0F
+        }
+
```

Comment:
I think we also have to set a `contentDescription` here. The Story Specification says: 

> Meaningful icons like the "tendency" icons (red arrow up etc.) should have image/icon descriptions with their meaning (respective text from help/info dialog content legend)

Reply:
I added the content description